src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/network/MixinClientPlayNetworkHandler.java

```diff
@@ -82,27 +78,6 @@ private void onChunkDeltaUpdateEnd(ChunkDeltaUpdateS2CPacket packet, CallbackInf
         ChunkUpdateFlag.chunkUpdate = false;
     }
 
-    @ModifyArgs(method = "onExplosion", at = @At(value = "INVOKE", target = "Lnet/minecraft/util/math/Vec3d;add(DDD)Lnet/minecraft/util/math/Vec3d;"))
-    private void onExplosionVelocity(Args args) {
-        double x = args.get(0);
-        double y = args.get(1);
-        double z = args.get(2);
-
-        if (ModuleAntiExploit.INSTANCE.getRunning() && ModuleAntiExploit.INSTANCE.getLimitExplosionStrength()) {
-            double fixedX = MathHelper.clamp(x, -10.0, 10.0);
-            double fixedY = MathHelper.clamp(y, -10.0, 10.0);
-            double fixedZ = MathHelper.clamp(z, -10.0, 10.0);
-
-            if (fixedX != x || fixedY != y || fixedZ != z) {
-                ModuleAntiExploit.INSTANCE.notifyAboutExploit("Limited too strong explosion",
-                        true);
-                args.set(0, fixedX);
-                args.set(1, fixedY);
-                args.set(2, fixedZ);
-            }
-        }
-    }
-
```

Comment:
Not sure if mine actually worked, but at least try to do something?

Reply:
Minecraft doesn't calculate explosions client side anymore, this exploit is fixed.

Reply:
Oh, Ok. not sure about if the server sends some big knockback or something (if that's covered in anti exploit) and the client takes it and crashes it.

Reply:
> Minecraft doesn't calculate explosions client side anymore, this exploit is fixed.

It doesn't calculate explosions client side anymore, but what does the client do? It handles the knockback.
And ExplodeBypass crashes without this, also, my implementation works.

> Oh, Ok. not sure about if the server sends some big knockback or something (if that's covered in anti exploit) and the client takes it and crashes it.

And this was true, I guess.